core/api/src/services/kratos/private.ts

```diff
@@ -30,16 +25,12 @@ export const getKratosPostgres = () =>
     connection: KRATOS_PG_CON,
   })
 
-export const toDomainSession = (session: KratosSession): Session => {
-  // is throw ok? this should not happen I (nb) believe but the type say it can
-  // this may probably be a type issue in kratos SDK
-  if (!session.identity) throw new InvalidIdentitySessionKratosError()
-  if (!session.expires_at) throw new MissingExpiredAtKratosError()
-
+export const toDomainSession = (session: KratosSession): MobileSession => {
   return {
     id: session.id as SessionId,
-    identity: toDomainIdentity(session.identity),
-    expiresAt: new Date(session?.expires_at),
+    identity: toDomainIdentity(session.identity as Identity),
+    expiresAt: new Date(session?.expires_at as string),
+    issuedAt: new Date(session?.issued_at as string),
```

Comment:
I checked the Session interface and expires_at and issued_are are already strings... cast is not necessary also can hide a crash because of conversion (Date)

Reply:
it can be undefined otherwise

![image](https://github.com/GaloyMoney/blink/assets/11156370/fe1ba140-4296-495e-8398-126934cc7f8c)

before I had this but it's more ugly

  if (!session.expires_at) throw new MissingExpiredAtKratosError()